tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -1505,7 +1505,7 @@ restapi/test_restapi_vxlan_ecmp.py:
   skip:
     reason: "Only supported on cisco 8102 T1"
     conditions:
-      - "not (asic_type in ['cisco-8000'] and 't1' in topo_type)"
+      - "not (platform in ['x86_64-8102_64h_o-r0', 'x86_64-8101_32fh_o-r0', 'x86_64-mlnx_msn4600c-r0']  and 't1' in topo_type)"
```

Comment:
Please double check the skip statement. `not t1 in topo_type` means the test will be skipped in Non-T1 topo, but the code specify the test to be run on T1.

Reply:
It should be run on 8102, 8101, 4600 T1s. I am not sure I understand the comment here.